pkg/arvo/neo/cod/std/src/fil/messenger.md

```diff
@@ -562,3 +563,273 @@ The `stem` is head-tagged with a `care:neo`. If the shrub’s dependent is liste
 All stems come with an `ever`, which contains version numbers for the dependency and its descendants. `%x` `stem`s come with a `pail` with the new state of the dependency shrub. The `%y` and `%z` `stem`s come with a `pail` for the dependency shrub, and a map containing all the relevant descendants of that dependency shrub. The `pith` keys in that map give us the locations of the descendants, and the `[=ever =mode =pail]` values give us their own version numbers, their `mode`, and their current state.
```

Comment:
Should stress here that an `ever` is not the same thing as a `care`. Maybe these should be different paragraphs.